posthog/tasks/utils.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+
+# NOTE: Keep in sync with bin/celery-queues.env
+class CeleryQueue(Enum):
```

Comment:
I added a `stats` queue for the priority consumer as well. So we can add that here or later.

Reply:
I'll add it here 👍